cmd/limactl/list.go

```diff
@@ -165,8 +182,46 @@ func listAction(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	w := tabwriter.NewWriter(cmd.OutOrStdout(), 4, 8, 4, ' ', 0)
-	fmt.Fprintln(w, "NAME\tSTATUS\tSSH\tVMTYPE\tARCH\tCPUS\tMEMORY\tDISK\tDIR")
+	types := map[string]int{}
+	archs := map[string]int{}
+	for _, instName := range instances {
+		inst, err := store.Inspect(instName)
+		if err != nil {
+			continue
+		}
+		types[inst.VMType]++
+		archs[inst.Arch]++
+	}
+
+	show, err := cmd.Flags().GetBool("show")
+	if err != nil {
+		return err
+	}
+
+	out := cmd.OutOrStdout()
+	w := tabwriter.NewWriter(out, 4, 8, 4, ' ', 0)
+
+	fmt.Fprint(w, "NAME\tSTATUS\tSSH")
+	cols := 5 // two for status, ssh
```

Comment:
and three for what?

Reply:
I think a `col` here means 8 characters, so 1 for `name`, 2 for `status` and 2 for `ssh` add up to 5. See line 218 on how it is used; that seems to be the only place the value is actually read and not updated.

Reply:
Maybe it would make more sense to count characters

Reply:
The "col" was a supposed to be a column in the TabWriter, defined as 8

```go
func NewWriter(output io.Writer, minwidth, tabwidth, padding int, padchar byte, flags uint) *Writer
```